accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -806,6 +806,8 @@ public interface Setup<State, SystemUnderTest>
         private Set<Setup<State, SystemUnderTest>> unknownWeights = null;
         @Nullable
         private Map<Predicate<State>, List<Setup<State, SystemUnderTest>>> conditionalCommands = null;
+        @Nullable
+        private Map<Predicate<State>, List<Pair<Setup<State, SystemUnderTest>, Integer>>> conditionalCommandsKnownWeights = null;
```

Comment:
this is here so we can disable repair when there are down nodes while running with Accord.  `nodetool repair --force` is timing out and looks to be durability scheduling related according to the logs.  I disabled running repairs when there are downed nodes and now we are mostly stable